jetstream/analysis.py

```diff
@@ -272,9 +275,8 @@ def calculate_metrics(
             )
         else:
             logger.info(
-                "Executing query for %s (%s)",
-                self.config.experiment.normandy_slug,
-                period.value,
+                f"Executing query for {self.config.experiment.normandy_slug}"
+                f"({period.value}) ({metric.name})"
```

Comment:
Just wondering if it makes sense to also log the metric slug separately. Might come in handy on the Looker dashboard to filter and see errors related to a specific metric? Doesn't have to be as part of this PR though

Reply:
I do have this separated out in the exception logging later on in `calculate_metrics`, so I think that covers what you're describing. But let me know if you think it would still be useful here (and perhaps in other logs where I've added the metric to the message).